keras_cv/bounding_box/converters.py

```diff
@@ -354,6 +356,22 @@ def convert_format(
         dtype: the data type to use when transforming the boxes.  Defaults to
             `tf.float32`.
     """
+    if isinstance(boxes, dict):
+        boxes["boxes"] = convert_format(
```

Comment:
Do we want the API to accept dict?
it's not bad to do `convert_format(y["boxes"], ...)` TBH.
Also if we do want to make the behavior change, then need to update the docstring.

Reply:
lets update the docstring, I personally think its trivial to support so we may as well.  Its a nice convenience.